rust/worker/src/execution/orchestration/compact.rs

```diff
@@ -503,6 +503,8 @@ impl Component for CompactOrchestrator {
 // ============== Handlers ==============
 #[async_trait]
```

Comment:
The orchestrator run() method could be a consumer of reply functionality, are we planning to do that in a seperate PR? Can we cut a task if not

Reply:
yes, was planning on a separate PR, will update the tracker